StudyHub/StudyHub/Controllers/AuthController.cs

```diff
@@ -9,30 +9,48 @@ namespace StudyHub.Controllers;
 public class AuthController : ControllerBase
 {
     private readonly IAuthService _authService;
+    private readonly IPasswordService _passwordService;
 
-    public AuthController(IAuthService authService)
+    public AuthController(IAuthService authService, IPasswordService passwordService)
     {
         _authService = authService;
+        _passwordService = passwordService;
     }
 
-    [HttpPost("[action]")]
-    public async Task<IActionResult> Register(RegisterUserDTO userDTO)
+    [HttpPost("sign-up")]
```

Comment:
#47 should remove this changes, so you can implement it if @Podgore don't want easy _points_